geomstats/geometry/general_linear.py

```diff
@@ -1,4 +1,9 @@
-"""The General Linear Group, i.e. the matrix group GL(n)."""
+"""
+This module exposes the `GeneralLinearGroup` class. 
+
+Note: This class is temporarily kept for compatibility purposes
+but will soon be renamed `GeneralLinear`.
```

Comment:
It is already renamed GeneralLinear, so we can delete this note.

Reply:
oops I got really mixed up in the rebase^^